config/puma.rb

```diff
@@ -53,6 +55,11 @@
 #   ActiveRecord::Base.establish_connection if defined?(ActiveRecord)
 # end
 #
+# before_fork do
```

Comment:
is this here so we can un-comment if we decide to use it?

Reply:
removed this for now